### ISSUE TITLE
When is ipex available for pytorch 2.3.0 ?

### ISSUE THREAD

**Issue by sriharshaj70** [intel/intel-extension-for-pytorch#624]
### Describe the issue

Currently ipex does not run with pytorch 2.3.0.
When can we expect to get the version compatible with pytorch 2.3.0 ?

**Comment by ChezRD** [comment]
What about XPU branch? Documentation says that +cpu is only for CPU branch

> GPU features are not included in CPU-only packages.

I have issues with ARC 750 crashing with pink squares randomly when large tasks performed (with Fedora 40 and docker), so I assumed that updated version can be viable to fix the issue, but didn't find any useful information. Also xpu-main specifically points to 2.1.30. And some python libraries already requires 2.2 and 2.3 versions of pytorch

**Comment by ChezRD** [comment]
@feng-intel, yes, I can make repo with my experients. I guess I will need to add list of installed packages on my Fedora distro, and provide docker container packages list into issue template?

> @ChezRD Could you start a new issue and give the code and steps about how to reproduce "crashing with pink squares randomly" issue ?
